wire-ios-request-strategy/Sources/Request Strategies/Conversation/ConversationEventProcessorTests.swift

```diff
@@ -115,7 +115,19 @@ class ConversationEventProcessorTests: MessagingTestBase {
 
     // MARK: - MLS conversation member leave
 
-    func test_UpdateConversationMemberLeave_WipesMLSGroup() {
+    func test_UpdateConversationMemberLeave_WipesMLSGroup_WithProtocolMLS() {
+        internalTest_UpdateConversationMemberLeave_WipesMLSGroup(messagingProtocol: .mls, shouldWipe: true)
+    }
+
+    func test_UpdateConversationMemberLeave_WipesMLSGroup_WithProtocolMixed() {
+        internalTest_UpdateConversationMemberLeave_WipesMLSGroup(messagingProtocol: .mixed, shouldWipe: true)
+    }
+
+    func test_UpdateConversationMemberLeave_DoesntWipeMLSGroup_WhenProtocolIsNotMLS() {
+        internalTest_UpdateConversationMemberLeave_WipesMLSGroup(messagingProtocol: .proteus, shouldWipe: false)
+    }
+
+    func internalTest_UpdateConversationMemberLeave_WipesMLSGroup(messagingProtocol: MessageProtocol, shouldWipe: Bool) {
```

Comment:
You could do that ⬇️ and add the file and line to the assert
```suggestion
    func internalTest_UpdateConversationMemberLeave_WipesMLSGroup(messagingProtocol: MessageProtocol, shouldWipe: Bool, file: StaticString = #file, line: UInt = #line) {
```